### DISCUSSION TITLE
This can make LTMEAT Flood actually viable

### DISCUSSION THREAD

**Discussion by PenguinLord561** [3kh0/ext-remover#924] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
When doing ltmeat flood after a few minutes the extension starts working again, reloading the page and blocking it. (for securly)
but what i have noticed is that when it shows the "Changes you made may not be saved" pop-up and you press cancel, it does not block it (until you reload the page). This works on sites like arras.io
my question is, is there a way to show that pop-up on say discord?

**Comment by AshtonDavies** [comment]
I cannot emphasize enough to get you guys to follow the instructions.

**Reply by PenguinLord561** [reply]
@AshtonDavies ? Wait am I being stupid?

**Reply by AshtonDavies** [reply]
Not really your fault, no. You're probably reading the readme, which is out of date, and most people look in the discussion and are supposed to use the Chrose link so they can get the right amount of tabs for their Chromebooks. I think that's usually the reason it doesn't last, but your extension also might not work permanently with this line Lightspeed and particular ones do.

Simple solution: If the effect doesn't stick, add more to the folder.
